fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1716,6 +1720,21 @@ class FMCMainDisplay extends BaseAirliners {
         }
     }
 
+    updateMinimums() {
+        const inRange = this.shouldTransmitMinimums();
+        const MDAssm = inRange && this.perfApprMDA ? Arinc429Word.SignStatusMatrix.NormalOperation : Arinc429Word.SignStatusMatrix.NoComputedData;
+        const DHssm = inRange && this.perfApprDH ? Arinc429Word.SignStatusMatrix.NormalOperation : Arinc429Word.SignStatusMatrix.NoComputedData
+
+        this.arincMDA.setBnrValue(this.perfApprMDA ? this.perfApprMDA : 0, MDAssm, 17, 131072, 0);
+        this.arincDH.setBnrValue(this.perfApprDH ? this.perfApprDH : 0, DHssm, 16, 8192, 0);
+    }
+
+    shouldTransmitMinimums() {
+        const phase = getFlightPhaseManager().phase;
+        //Only transmit when in Cruise or later phase and within 250NM of destination
+        return (phase > FmgcFlightPhase.Cruise || (phase === FmgcFlightPhase.Cruise && this.flightPlanManager.getDistanceToDestination(FlightPlans.Active) < 250));
```

Comment:
In fact this doesn't really align with the comment, this would be transmit in FMGC phase is greater than Cruise OR transmit if FMGC phase is Cruise AND within (but not including) 250nm of destination